src/benchmarks/micro/runtime/System.Reflection/Invoke.cs

```diff
@@ -24,6 +24,15 @@ public class Invoke
         private static MethodInfo s_method_nullableInt;
         private static ConstructorInfo s_ctor_int_string_struct_class;
         private static ConstructorInfo s_ctor_NoParams;
+
+#if NET8_0_OR_GREATER
```

Comment:
For my own understanding, does this macro mean the APIs were introduced in .NET 8?

Reply:
Correct